style.css

```diff
@@ -0,0 +1,1769 @@
+    align-content: center;
+    justify-content: center;
+    align-items: center;
+    gap: 10px;
+  }
+
+  .languageskill1 {
+    display: flex;
+    flex-direction: column;
+    gap: 12px;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .language {
+    height: 24px;
+    font-family: 'InaiMathi', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    margin-bottom: 13px;
+  }
+
+  .languagelist {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+  }
+
+  .languages {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+  }
+
+  .languageitem {
+    padding: 0 12px;
+  }
+
+  .rectangle {
+    width: 18px;
+    height: 18px;
+  }
+
+  .item1,
+  .item2,
+  .item3,
+  .item4 {
+    padding: 0 12px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 43px;
+  }
+
+  .formsection {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 68px;
+  }
+
+  .formtext {
+    text-align: center;
+    color: #262626;
+    width: auto;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .formcontainter {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0;
+    gap: 24px;
+    width: 100%;
+  }
+
+  .forminputs {
+    display: block;
+  }
+
+  input {
+    width: 100%;
+    height: 48px;
+    border-top: 0;
+    border-left: 0;
+    border-right: 0;
+    border-bottom: solid 0.6px #ccc;
+  }
+
+  textarea {
+    width: 100%;
+    height: 114px;
+    border: none;
+    background: #fbf8f7;
+    color: #3c3a39;
+    padding: 12px;
+  }
+
+  textarea::placeholder {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    color: #3c3a39;
+    width: 100%;
+  }
+
+  li {
+    list-style: none;
+  }
+
+  .formitems {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 24px;
+    margin-bottom: 6rem;
+  }
+
+  .submitbutton {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px;
+    gap: 10px;
+    background: #ff6b00;
+    width: 131px;
+    height: 48px;
+    border: none;
+    color: #ffff;
+    text-align: center;
+    align-content: center;
+    justify-content: center;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+  }
+
+  .iconscontainer2 {
+    display: flex;
+  }
+
+  .iconscontainer1 {
+    display: none;
+  }
+
+  .icons3 {
+    display: none;
+  }
+
+  /* modal view */
+
+  .modal {
+    display: none;
+    position: fixed;
+    z-index: 999;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+    background-color: rgb(0, 0, 0);
+    background-color: rgba(0, 0, 0, 0.4);
+  }
+
+  .modal-content {
+    margin: 0;
+    padding: 30px;
+  }
```

Comment:
- [ ] For the desktop version of the pop-up window it is too close to the right border, kindly center the pop-up window to look similar to the Figma template.
- [ ] For the mobile version the pop-up window is too close to the borders, please 🙏🏽 center it to look similar to the Figma template.

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/202827782-cac021ed-5d80-47af-ae48-2f937f6bf34c.png) | ![image](https://user-images.githubusercontent.com/58642949/202827796-dd05f348-25d4-46d8-b1a5-3cb6304cd3cf.png) |
| ![image](https://user-images.githubusercontent.com/58642949/202827966-06d8d499-8f0c-4ed5-b500-12c18d351901.png) | ![image](https://user-images.githubusercontent.com/58642949/202827975-5be1f290-7e10-4f26-9c4f-d293123005fa.png) |